src/rdb/rdb_tx.c

```diff
@@ -388,6 +388,21 @@ rdb_tx_append(struct rdb_tx *tx, struct rdb_tx_op *op, bool is_critical)
 	return 0;
 }
 
+void
+rdb_tx_discard(struct rdb_tx *tx)
```

Comment:
[Nit] We might want to add a Doxygen comment in a subsequent revision or PR.

Reply:
will do now since I have to make other changes to the PR